Controller/ChangePasswordController.php

```diff
@@ -86,6 +86,15 @@ public function changePasswordAction(Request $request)
 
         return $this->render('@FOSUser/ChangePassword/change_password.html.twig', array(
             'form' => $form->createView(),
+            'data' => $this->setUserData(),
         ));
     }
+
+    /*
+     * Function to override to send data to the template
```

Comment:
In German, it's actual called `Funktion` :D

A method is a kind of function used in the context of a class/object.